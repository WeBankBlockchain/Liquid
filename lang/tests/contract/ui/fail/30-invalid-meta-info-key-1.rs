use liquid_lang as liquid;

#[liquid::contract(version = "1.0.0-rc2", invalid_key = "whatever")]
mod noop {
    #[liquid(storage)]
    struct Noop {}

    #[liquid(methods)]
    impl Noop {
        pub fn new(&mut self) {}

        pub fn noop(&self) {}
    }
}

fn main() {}
