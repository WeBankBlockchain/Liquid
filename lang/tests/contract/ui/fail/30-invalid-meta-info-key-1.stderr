error: unknown parameter: `invalid_key`
 --> $DIR/30-invalid-meta-info-key-1.rs:3:43
  |
3 | #[liquid::contract(version = "1.0.0-rc1", invalid_key = "whatever")]
  |                                           ^^^^^^^^^^^
